packages/x-components/package.json

```diff
@@ -53,14 +53,17 @@
     "prepublishOnly": "npm run build",
     "test:unit": "jest --silent && npm run test:unit-cypress",
     "test:unit-jest": "jest",
-    "test:unit-cypress": "cypress run-ct",
+    "test:unit-cypress": "cypress run --component",
     "test:unit-coverage": "jest --coverage",
     "test:unit-watch": "jest --watch",
-    "test:e2e": "vue-cli-service test:e2e",
+    "test:e2e": "vue-cli-service test:e2e --e2e --browser chrome",
+    "test:e2e:firefox": "vue-cli-service test:e2e --e2e --browser firefox",
```

Comment:
**nitpick**: we are using the `--e2e` parameter to be exhaustive and explicit, but it is not necessary as it is the default behaviour.

```suggestion
    "test:e2e": "vue-cli-service test:e2e --browser chrome",
    "test:e2e:firefox": "vue-cli-service test:e2e --browser firefox",
```

Reply:
I don't think it is the default value. If I run cypress without that flag I see this screen, allowing me to choose the kind of tests to run:
![image](https://user-images.githubusercontent.com/68222542/210072288-989c736a-ab00-4be1-b931-d324ddc74088.png)

Reply:
So they lie in their own [documentation](https://docs.cypress.io/guides/guides/command-line#Options-1) 😞 

<img width="682" alt="image" src="https://user-images.githubusercontent.com/6247440/210075280-12c22843-895e-448f-a6f1-6ec7612229ff.png">